src/main/java/at/hannibal2/skyhanni/utils/LorenzUtils.kt

```diff
@@ -269,6 +269,16 @@ object LorenzUtils {
             )
     }
 
+    fun GuiEditSign.isMousematSign(): Boolean {
+        if (this !is AccessorGuiEditSign) return false
+
+        val tileSign = (this as AccessorGuiEditSign).tileSign
+        return (
+            tileSign.signText[1].unformattedText.removeColor() == "Set Yaw Above!" &&
+                tileSign.signText[2].unformattedText.removeColor() == "Set Pitch Below!"
+            )
+    }
```

Comment:
I know the othger method is in here but nothing new should be added to LorenzUtils. Would also be good if a new home was found for the other method

Reply:
What about creating a new Object `SignUtils` and add the methods `setTextIntoSign`, `addTextIntoSign`, `isRancherSign` and `isMousematSign`. You can also outsource some code from `BetterSignEditing` and only leave the function calls in there

Reply:
I like this idea

Reply:
this 845ff5ddcf36739ef893215b351deb29cec4e994 should solve it